src/popup.tsx

```diff
@@ -70,6 +71,12 @@ function IndexPopup() {
           checked={automaticallyDetect}
           onChange={setAutomaticallyDetect}
         />
+        <CheckboxInputField
+          label="Always start with options"
```

Comment:
If we decide to use this in favor of the dropdown / secondary "start with options" (I think we should), maybe rewording it for more clarity, since we have more space in this UI vs the button. Personally, I think the default behavior would be best to not autostart, and then the setting lets you opt in to auto starting. Then something like:

> Autostart workspaces